platforms/nuget/build-jinja.py

```diff
@@ -1,32 +1,78 @@
-import os
+import os, argparse
+from pathlib import Path
+from shutil import copyfile
 from jinja2 import Environment, FileSystemLoader
 PATH = os.path.dirname(os.path.abspath(__file__))
 TEMPLATE_ENVIRONMENT = Environment(
     autoescape=False,
     loader=FileSystemLoader(os.path.join(PATH, 'templates')),
     trim_blocks=False)
 
+VERSION_PATH = str(Path(PATH).parents[1]) + "/modules/core/include/opencv2/core/version.hpp"
+version_file = Path(VERSION_PATH).read_text()
+
+version_major = int(version_file.split("CV_VERSION_MAJOR")[1].split("\n")[0])
+version_minor = int(version_file.split("CV_VERSION_MINOR")[1].split("\n")[0])
+version_revision = int(version_file.split("CV_VERSION_REVISION")[1].split("\n")[0])
+
+version_string = str(version_major) + "." + str(version_minor) + "." + str(version_revision)
```

Comment:
Instead of 
version_string = str(version_major) + "." + str(version_minor) + "." + str(version_revision)
you should use formattable string as:
version_string = f'{version_major}.{version_minor}.{version_revision}'

Reply:
This has now been fixed.